.travis.yml

```diff
@@ -4,7 +4,7 @@ node_js:
 sudo: required
 dist: trusty
 addons:
-  firefox: latest
+  firefox: 53.0
```

Comment:
Ace. Maybe add comment to revisit in a bit when test cafe updates - because we don't want to be here for long.